docs/modules/ROOT/pages/visual-tour.adoc

```diff
@@ -326,7 +331,7 @@ Therefore, this feature is only useful if you have a causal cluster.
 
 | `enableMultiStatementMode`
 | `true`
-| Allows you to write and edit multi-line queries (use semicolon to separate statements).
+| Allows you to write and edit multi-line queries (use a semicolon to separate statements).
```

Comment:
```
| Allows you to write and edit multi-line queries (use a `;` (semicolon) to separate statements).
```

Reply:
It's either "a semicolon (`;`)" or "`;` (a semicolon)", because "a" goes with a noun. I actually think that we don't have to explain what a semicolon is as everybody knows.